File: kaioken/index/test/fixtures/langrepo/sample.go
package sample

import "fmt"

// Add returns the sum of a and b.
// It exists so the fixture has a two-line doc comment.
func Add(a, b int) int {
	return a + b
}

func unexported(x int) int {
	return x
}

// Shape is the interface every renderable implements.
type Shape interface {
	Area() float64
}

// Rect is a rectangle.
type Rect struct {
	W float64
	H float64
}

// Area implements Shape.
func (r Rect) Area() float64 {
	return r.W * r.H
}

const Pi = 3.14159

var Registry = map[string]Shape{}

type Alias = string

// Kind classifies a declaration.
type Kind string

const (
	KindFunc Kind = "func"
	KindType Kind = "type"
)

File: kaioken/index/test/fixtures/langrepo/sample.py
"""Module docstring, not attached to any declaration."""


def add(a, b):
    """Return the sum of a and b."""
    return a + b


def _private(x):
    return x


class Rect:
    """A rectangle."""

    def area(self):
        """Compute the area."""
        return self.w * self.h

    def _hidden(self):
        return None


MAX_SIZE = 4096
